docs/assets/includes/client-side-tool-instructions.md

```diff
@@ -1,6 +1,8 @@
-<!--instructions-start-->
+<!--instructions-ui-setup-start-->
 1.  [Enable {{ page.meta.tool_name }}](../../repositories-configure/configuring-code-patterns.md) and configure the corresponding code patterns on your repository **Code patterns** page.
+<!--instructions-ui-setup-end-->
```

Comment:
This step is actually required whenever we run the Codacy Analysis CLI with the project or account API token, since the CLI connects to Codacy and uses the tool and code pattern settings that are currently defined on the repository.

We do have to make this step generic because we're now talking about any tool instead of just a specific client-side tool.

Reply:
Sounds good. I took a properly set-up configuration as a given on a "generic" run of the CLI.

I'll think something up to make this step more general.